Python/bytecodes.c

```diff
@@ -4235,6 +4233,7 @@ dummy_func(
             unused/1 +
             unused/2 +
             _CALL_BUILTIN_O +
+            _POP_TWO +
```

Comment:
```suggestion
            POP_TOP +
            POP_TOP +
```
The reason is that it's easier to optimize this way.